dotcom-rendering/src/model/enhance-H3s.ts

```diff
@@ -0,0 +1,92 @@
+import { JSDOM } from 'jsdom';
+
+const isFalseH3 = (element: CAPIElement): boolean => {
+	if (!element) return false;
+	// Checks if this element is a 'false h3' based on the convention: <p><strong><H3 text</strong></p>
+	if (
+		element._type !== 'model.dotcomrendering.pageElements.TextBlockElement'
+	) {
+		return false;
+	}
+	const frag = JSDOM.fragment(element.html);
+	if (!frag || !frag.firstElementChild) return false;
+	const html = frag.firstElementChild.outerHTML;
+	// The following things must be true for an element to be a faux H3
+	const hasPwrapper = frag.firstElementChild.nodeName === 'P';
+	const containsStrongtags =
+		frag.firstElementChild.outerHTML.includes('<strong>');
+	const doesNotContainLinks =
+		!frag.firstElementChild.outerHTML.includes('<a>');
+	const startStrong = html.startsWith('<p><strong>');
+	const endsStrong = html.endsWith('</strong></p>');
+
+	return (
+		hasPwrapper &&
+		containsStrongtags &&
+		doesNotContainLinks &&
+		startStrong &&
+		endsStrong
+	);
+};
+
+const extractH3 = (element: CAPIElement): string => {
+	// Extract the text based on the convention: <p><strong><H3 text</strong></p>
+	const textElement = element as TextBlockElement;
```

Comment:
1. Could the comment be moved to a JSDoc style?

2. Why does the function not take a TextBlockElement, but rather does a type assertion? I can help with creating a type-guard and we have other examples in the enhancers doing exactly this

3. You're doing the `isFalseH3` check twice. It might be better to create a new type-guard especially for this and I'm happy to pair on it.

Reply:
Happy to pair on this today if you have a little time? Anna is not working today.